web/src/lib/components/pages/record/audio-web.ts

```diff
@@ -111,68 +110,63 @@ export default class AudioWeb {
     this.volumeCallback = cb;
   }
 
-  init() {
-    if (this.initPromise) {
-      return this.initPromise;
+  async init(): Promise<void> {
```

Comment:
We still need a check here to see if we are already initialized. I agree we will still have an underlying race if the user clicks record twice really fast, and we should solve that in a different bug. But with this change, we initialize a new audio context every time the user presses record, which is unnecessary.